starknet_devnet/util.py

```diff
@@ -243,3 +244,26 @@ def get_fee_estimation_info(tx_fee: int, gas_price: int):
 def warn(msg: str, file=sys.stderr):
     """Log a warning"""
     print(f"\033[93m{msg}\033[0m", file=file)
+
+
+class SuppressLogger:
+    """ctx mng to suppress logger"""
```

Comment:
Can you use full words here? Is that supposed to be `Context manager`?

Reply:
fixed